tests/ui-toml/toml_import_rename/conf_import_rename.rs

```diff
@@ -0,0 +1,14 @@
+#![allow(unused_imports, dead_code)]
+#![warn(clippy::import_rename)]
```

Comment:
Is this required? I think `disallowed_method` is warn-by-default but since the default list is empty it just never triggers. Seems like the same thing would work here.

Reply:
The restriction and nursery categories are all `allow` by default I think, I did try but it gave me no output :shrug: